tests/frontend/test-class-frontend-woocommerce-shop-page.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * @package WPSEO\Tests\Frontend
+ */
+
+/**
+ * Unit Test Class.
+ */
+class WPSEO_Frontend_WooCommerce_Shop_Page_Test extends WPSEO_UnitTestCase {
+	/**
+	 * Tests for original title on non-shop page.
+	 */
```

Comment:
It would be nice to have `@covers` here, to make clear what this test is covering.